src/views/admin/AllOperations.vue

```diff
@@ -0,0 +1,189 @@
+<template>
+    <button class="flex items-center mb-4 navigation-link mt-12" @click="back">
+        <i class="fas text-xl fa-chevron-left"></i>
+        <span class="font-bold text-sm ml-1">Back</span>
+    </button>
+    <div class="flex flex-col items-center justify-center w-full">
+        <h1 class="text-4xl font-semibold text-blue-800 mb-10">Operations</h1>
+        <h2 class="text-xl text-gray-700">
+            In this dashboard, you find all operations in the system. You may watch them for displaying them in your personal dashboard
+            <router-link id="routeWelcomePage" :to="{ name: 'welcome' }" class="navigation-link hover:cursor-pointer hover:underline">
+                here
+            </router-link>
+            .
+        </h2>
+    </div>
+    <div v-if="busy" class="mx-auto">
+        <loading-spinner />
+    </div>
+    <div v-else class="flex flex-col items-center justify-center w-full mt-10">
+        <button v-if="!gotOps" class="btn btn-blue-primary p-4 mt-10" @click="requestData">Request Operations</button>
+        <dashboard-component
+            v-else
+            identifier="all"
+            class="w-full"
+            :watched-operations="watchedOperations"
+            :operations="operations"
+            :role="role"
+            :is-archive="true"
+            title="All Operations"
+        />
+    </div>
+</template>
+<script lang="ts">
+    import { useStore } from "@/use/store/store";
+    import { ref, onBeforeMount } from "vue";
+    import LoadingSpinner from "@/components/common/loading/Spinner.vue";
+    import { checkPrivilege } from "@/use/helpers/PermissionHelper";
+    import { Role } from "@/entities/Role";
+    import DashboardComponent from "@/components/common/dashboard/DashboardComponent.vue";
+    import Operation, { ApprovalList } from "@/api/api_models/operation_management/Operation";
+    import { UC4Identifier } from "@/api/helpers/UC4Identifier";
+    import { OperationStatus } from "@/api/api_models/operation_management/OperationState";
+    import Router from "@/use/router/";
+    import OperationManagement from "@/api/OperationManagement";
+    import GenericResponseHandler from "@/use/helpers/GenericResponseHandler";
+
+    export default {
+        name: "AllOperationsPage",
+        components: {
+            LoadingSpinner,
+            DashboardComponent,
+        },
+
+        async beforeRouteEnter(_from: any, _to: any, next: any) {
+            const response = await checkPrivilege(Role.ADMIN);
+
+            if (response.allowed) {
+                return next();
+            }
+            if (!response.authenticated) {
+                return next("/login");
+            }
+
+            return next("/redirect");
+        },
```

Comment:
didnt we add this to the router index long long ago?

Reply:
Absolutely, that's a copy&paste relict. We may have to refactor the whole application to find more of them. Removed in https://github.com/upb-uc4/ui-web/pull/785/commits/b8ab13040a164069e0ee9199787c80e4bc80a5ad